cmd/root.go

```diff
@@ -38,3 +43,33 @@ func NewRootCmd() *cobra.Command {
 
 	return cmd
 }
+
+func runE(cmd *cobra.Command) error {
+	logLevelFlag, _ := cmd.Flags().GetString("loglevel")
+
+	if err := addLogger(logLevelFlag); err != nil {
+		return err
+	}
+	return nil
+}
+
+func addLogger(logLevelFlag string) error {
+	if err := validation.ValidateChoice(
+		logLevelFlag,
+		[]string{"DEBUG", "INFO", "WARNING"},
+		"loglevel",
+	); err != nil {
+		return err
+	}
+	level, err := log.ParseLevel(logLevelFlag)
+	if err != nil {
+		return err
+	}
+	log.SetLevel(level)
+	log.SetOutput(os.Stdout)
```

Comment:
What do you think about writing to `os.Stderr`, just like the Python client does?

Reply:
Nice catch